blivet/tasks/availability.py

```diff
@@ -298,6 +298,7 @@ def available_resource(name):
 MKFS_XFS_APP = application("mkfs.xfs")
 MKNTFS_APP = application("mkntfs")
 MKREISERFS_APP = application("mkreiserfs")
+MLABEL_APP = application("mlabel")
```

Comment:
`fatlabel` doesn't have the ability to set UUIDs yet, see https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=563746 and its source at https://github.com/dosfstools/dosfstools/blob/820c2f90726db0468e0a684a5dc500fbde66466f/src/fatlabel.c.